ui/util/api/queries.js

```diff
@@ -30,10 +30,11 @@ export async function policiesData({ query }) {
     endpoints.policies.withIds(query.id__in),
     endpoints.topics.withIds(query.requirements__topics__id__in),
     endpoints.policies.fetch(Object.assign({ ordering: 'policy_number' }, query)),
+    endpoints.policies.withOmbPolicyIds(query.id__in),
```

Comment:
I don't think this'll get you the results you're looking for. There were five different queries happenings here, four of which were looking up ids (to grab names to display to the user) but the fifth (`policies.fetch`) grabbed a filtered list of policies, depending on the user's query. By adding this line, we're extending the lookup search (i.e., we can find more policy _names_) but not the policy search (i.e. we won't get additional matches).

Minor: if we're keeping this, it'd probably make sense to keep the id-lookup queries together (meaning we should move this line up).

Reply:
I missed this. Fixing.